test/snowcrashtest.h

```diff
@@ -146,6 +146,27 @@ namespace snowcrashtest {
             return signature;
         }
     };
+
+    /**
+     * \brief Helper to test source maps
+     */
+    struct SourceMapHelper {
+
+        /**
+         * If 'nth' is not given, sourceMap is of size 1 and checks the 0th element
```

Comment:
Basically, it checks a row in the sourceMap. If the row's number is given, it checks that particular row. Otherwise, it checks that the number of rows is 1, and checks the first row.